config/settings.yml

```diff
@@ -1238,6 +1238,7 @@
       :performance_rollup_purging_start_delay: 5.minutes
       :policy_events_purge_interval: 1.day
       :poll: 15.seconds
+      :queue_timeout_interval: 10.minutes
```

Comment:
I agree that this could probably be dropped so as not to extend a message's actual timeout from `msg_timeout` to `msg_timeout + 10.minutes`.

Maybe`1.minute` or `30.seconds` is enough of a delay that we're not doing it constantly but short enough that will not delay things that are waiting for the message to timeout.

Reply:
the job proxy dispatcher is queued every 15 seconds for reference... I know, bad example but still an example.